providers/aws/aws_provider.go

```diff
@@ -98,7 +100,8 @@ func (p *AWSProvider) InitService(serviceName string) error {
 	p.Service.SetName(serviceName)
 	p.Service.SetProviderName(p.GetName())
 	p.Service.SetArgs(map[string]string{
-		"region": p.region,
+		"region":  p.region,
```

Comment:
I have other branch with refactor this code. 
It's move to be `map[string]interface{}` 
I will changed this after merge